### ISSUE TITLE
namespace test failing with q35 on centos

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/runtime#1604]
CentOS q35 CI failing:

```
=== RUN   TestHostNetworkingRequested
--- FAIL: TestHostNetworkingRequested (0.01s)
    assertions.go:239: 
                          
	Error Trace:	network_test.go:99
        
	Error:      	Should be true
```

See https://github.com/kata-containers/runtime/pull/1587.

**Comment by jodh-intel** [comment]
Also seen here: https://github.com/kata-containers/runtime/pull/1599#event-2306049751.

**Comment by lifupan** [comment]
Hi @jodh-intel, 

I found the root cause is that the kernel in CentOS q35 is 3.10 which doesn't support the "nsfs",
and the "nsfs" isn't supported until  3.19 version. 
I'm curious when did we add the centos q35 test? Why this failed didn't appear until recently?

**Comment by jodh-intel** [comment]
Thanks @lifupan.

 The Jenkins q35 jobs were added on https://github.com/kata-containers/ci/pull/144, but I'm not sure if the date on that PR is the same as the time we actually started using them. Maybe @GabyCT can provide further info?

Rather than ignoring on centos, I think this is another use-case for https://github.com/kata-containers/runtime/pull/1587. I'll add a new call to allow skipping a test if the kernel version is too old...
